ConsistencyManager/ConsistencyManager.swift

```diff
@@ -400,6 +399,18 @@ open class ConsistencyManager {
      Adds an update listener to the consistency manager.
      This listener will get notified of ALL changes posted to the consistency manager.
      This method must be called on the main thread.
+
+     ## Use Case
+
+     This is useful if you want to filter on changes to the consistency manager. For instance, you may want to listen to all added models of a certain class.
+     If a model is added, you could add it to an existing array (similar to predicates).
+     Or you could listen on a model which you have an ID for, but the model isn't in the system yet.
+
+     ## Performance
+
+     The `ConsistencyManagerUpdatesListener` methods are called on the main thread for every model updated in the system.
+     It's recommended to do as minimal processing as possible here so you don't block the main thread.
```

Comment:
Can't work out how to do that in docs and don't think it's completely worth it. But lmk if you know the syntax